src/c++/uds/src/uds/index-session.c

```diff
@@ -624,9 +624,10 @@ int uds_close_index(struct uds_index_session *index_session)
 	/* Wait for any current index state change to complete. */
```

Comment:
I can't figure out how to make a note there, but dit looks like Mike's change had a change around line 394 of index-session.c that got completely dropped.